appmap/test/data/expected.appmap.json

```diff
@@ -324,6 +324,7 @@
         {
           "name": "dump",
           "type": "function",
+          "comment": "# Serialize a Python object into a YAML stream.\n# If stream is None, return the produced string instead.",
```

Comment:
This seems too brittle to me. (What if the yaml package changed their doc?) Can we filter this instead?

Reply:
Good point, I'll do that.